### DISCUSSION TITLE
Change the default keybindings for mode transition in the next version.

### DISCUSSION THREAD

**Discussion by pit-ray** [pit-ray/win-vind#118] [News]
Everyone says `<esc-*>` keybindings conflict with other applications in Issues.

Therefore, I plan to change the default keybindings to the following in the next major update.

- Conventional keybindings
GUI normal mode: `<esc-left>`  
Editor normal mode: `<esc-right>` 

- New
GUI normal mode: `<ctrl-]>` 
Editor normal mode: `<ctrl-[>` 

If you have better ideas than this change, please let me know.

**Comment by gilbh** [comment]
Looks good to me.

**Comment by jjensen** [comment]
I very often hit <ctrl-[> in Vim instead of reaching for Escape. I'm not sure your proposal would preserve that behavior.

I've been using Ctrl-Space to go back and forth between to_gui_normal and to_insert. I very much like the convenience of it. I am also not happy with the binding, because it breaks, say, auto-completion popups in Visual Studio and Rider. Maybe I should go for Windows-Space instead...

**Reply by pit-ray** [reply]
This binding is just default, so you can change ofcourse. If the binding conflicts to the original Vim, use a suppression option of bindings for Vim.

**Reply by pit-ray** [reply]
Anyway, I'll conduct a beta-test, so please let me know your user experiences then.
